krabs/krabs/ut.hpp

```diff
@@ -19,6 +19,14 @@ namespace krabs { namespace details {
     struct ut {
 
         typedef krabs::provider<> provider_type;
+		
+		struct FilterSettings
+		{
+			std::vector<unsigned short> m_OrigEventIds;
```

Comment:
nit: coding style in this project is to avoid hungarian notation. Consider `providerFilterEventIds`.

Reply:
`provider_filter_event_ids_` is the format to use for instance variables on the C++ classes.

Reply:
Done